### ISSUE TITLE
Get author name of posts need lots of requets

### ISSUE THREAD

**Issue by wb14123** [not-an-aardvark/snoowrap#40]
For example:

```
r.get_hot().map(x => x.author.name).then(console.log)
```

While I inspect the network I found this will request every user's info for each post's author. But the author name is already exist in the response of Reddit API. Is there any way to avoid these requets?

**Comment by wb14123** [comment]
Sorry. It seems to be the problem of Vue.js. It seems to fetch all the properties of the object, so it need to request Reddit API. I know it may irrelevant about this project, but I'd like to ask do you know how to avoid this?

**Comment by not-an-aardvark** [comment]
Awhile ago I had been considering a `proxies: false` config option that would disable this behavior; I think that would solve the issue. (At the moment this behavior is enabled in all environments that support the `Proxy` object, but there's no way to actively disable it.)

**Comment by not-an-aardvark** [comment]
This should be fixed in v1.7.0; to avoid this, you can use:

```js
r.config({proxies: false});
```

**Comment by wb14123** [comment]
Thanks!
